src/app/beer_garden/api/http/handlers/v1/garden.py

```diff
@@ -182,8 +196,9 @@ async def get(self):
           - Garden
         """
         permitted_gardens: QuerySet = self.permissioned_queryset(Garden, GARDEN_READ)
-
-        response = GardenSchema(strict=True, many=True).dumps(permitted_gardens).data
+        response = self.validate_or_raise(
+            GardenSchema, permitted_gardens, method="dumps", many=True
+        )
```

Comment:
This, for example, could be replaced by

```python
response = GardenSchema(strict=True, many=True).dumps(permitted_gardens).data
```
because we can be sure that the result of `permissioned_queryset` won't throw a validation error. But that's really no shorter. Perhaps the function can be aliased to `validate` to make it clearer...